generators/craft3/templates/config/environment-label.php

```diff
@@ -1,8 +1,10 @@
 <?php
 
 return [
-  'showLabel' => filter_var(getenv('APP_DEV_MODE'), FILTER_VALIDATE_BOOLEAN),
-  'label' => 'Testing/Staging Environment',
+  'showLabel' => filter_var(getenv('APP_ENV_LABEL'), FILTER_VALIDATE_REGEXP,  array(
+     'options' => array('regexp' => "/.*/")
```

Comment:
Does this match blank strings? the `*` usually means "zero or more"

Reply:
It does match blank strings. I tested this with:

- No value set
- Value set to blank string
- Value set to non-blank string

Reply:
Sorry, I was a bit confused, but I just answered my question. You mentioned, "it doesn't show up if APP_ENV_LABEL is missing or empty" and I thought an empty string still made `showLabel` truthy, but it doesn't.